build.sbt

```diff
@@ -163,16 +163,15 @@ lazy val commonSettings =
         "-language:existentials",
         "-Ypartial-unification",
         //Flink image is available only for jdk8
```

Comment:
Let's write `we remain JDK8 compatible until 0.4.0 release` or sth like that

Reply:
By mistake missed this one flink's reference - replaced with other message. I don't want to specify if it will be exact 0.4.0 version here, because we haven't discussed it yet. Maybe we should check before how compatibility with java8 is broken in other similar projects?